docs/command_line_reference.rst

```diff
@@ -913,21 +913,21 @@ The required format is ``key`` ":" ``value``. You can choose ``key`` and  ``valu
 
  ::
 
-   esrally race --track=pmc --user-tag="intention:github-issue-1234-baseline,gc:cms"
+   esrally race --track=pmc --user-tags="intention:github-issue-1234-baseline,gc:cms"
 
 You can also specify multiple tags. They need to be separated by a comma.
 
 **Example**
 
  ::
 
-   esrally race --track=pmc --user-tag="disk:SSD,data_node_count:4"
+   esrally race --track=pmc --user-tags="disk:SSD,data_node_count:4"
 
 
 
-When you run ``esrally list races``, this will show up again::
+Wesrally list raceshen you run ``esrally list races``, this will show up again::
```

Comment:
I think this was introduced by accident?

Reply:
unintended, thank you for the catch 🙏